### PR TITLE
Spin: fix for widget rendering with <13 items

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1486
[Problem] Number Spinner with < 13 items not rendering correctly
[Solution]
     - method for roll items has been changed
     - additional manual tests
[Manual test]
   tests/manualTest/numberPicker/number-0-11.html
   tests/manualTest/numberPicker/number-0-12.html
   tests/manualTest/numberPicker/number-0-2.html
   tests/manualTest/numberPicker/number-0-3.html
   tests/manualTest/numberPicker/number-0-8.html
   tests/manualTest/numberPicker/number-0-9.html

    
Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

**Review by lslachciak — COMMENTED** [review]
please fix directory name numeberPicker --> numberPicker

File: src/js/core/widget/core/Spin.js [hunk 1]
```diff
@@ -193,14 +193,20 @@
 			var self = this,
 				direction = delta > 0 ? 1 : -1,
 				borderItem,
-				newItemToPlace;
+				newItemToPlace,
+				halfOfFreeCarouselItems = Math.round((self._numberOfCarouselItems - self._ui.items.length) / 2);
```
Comment (DohyungLim) [inline src/js/core/widget/core/Spin.js:200]: did the issue continue even before halfOfFreeCarouselItems was applied?
or did it happen intermittently?

**Review by DohyungLim — COMMENTED** [review]
1 comment.

File: src/js/core/widget/core/Spin.js (hunk 1 above)
Comment (TomaszLukawskiSam) [reply src/js/core/widget/core/Spin.js:200]: The problem was always when number of spin elements was < NUMBER_OF_CAROUSEL_ITEMS. However, initially the NUMBER_OF_CAROUSEL_ITEMS was equal 5 or 7 while the number picker had 10+ values and the issue was not so visible.

**Review by lbardeli — APPROVED** [review]
LGTM

**Pull request merged**